src/reducers/fileTree.spec.tsx

```diff
@@ -693,7 +693,7 @@ describe(__filename, () => {
           currentMessageUid: '',
           messageMap,
         }),
-      ).toEqual({ path: file1, uid: global1 });
+      ).toEqual({ line: null, path: file1, uid: global1 });
```

Comment:
You shouldn't have had to update this test because all it's doing is checking that the *first* global message is discovered. I suggest changing the assertion to:

```ts
      expect(
        _getRelativeMessage({
          currentMessageUid: '',
          messageMap,
        }),
      ).toEqual(expect.objectContaining({ uid: global1 }));
```

It's not important to add `line` to the test assertions when not relevant because TypeScript covers that logic.